pyproject.toml

```diff
@@ -88,7 +88,7 @@ docs = [
     "mkdocs-get-deps>=0.2.0",
     "mkdocstrings>=0.29.1",
     "mkdocstrings-python>=0.29.1",
-    "linkchecker>=10.2.1",
+    "griffe==1.6.0"  # Required for hatch run docs:build to work.
```

Comment:
Do we need this? We don't use hatch afaik.

Reply:
sorry i updated the comment, we need this specifically for the mkdocstrings-python, the comment is not very clear